scripts/e2e.sh

```diff
@@ -8,7 +8,7 @@ setup_service java 1.8.222
 setup_service google-chrome-stable 89.0.4389.72-1
 
 export TEST_SUITE_TYPE="junit"
-export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2"
```

Comment:
Bacon scripts zip whatever directory is listed in this env variable and saves it to `TestResults.zip`.

Reply:
This ensures we capture `/build2/reports/junit` still but also include the `/build2/screenshots` directory that contains the screenshots captured in these test runs.